src/components/common/RedactableValue.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FunctionComponent, useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTimes } from "@fortawesome/free-solid-svg-icons";
+import styled from "@emotion/styled";
+import { noop } from "../../utils";
```

Comment:
huh, didnt realise we have a `noop` helper
normally will just give a blank function